rakefile.rb

```diff
@@ -5,20 +5,34 @@
   raise "MSBuild not found"
 end
 
-nuget_command  = ".nuget/nuget.exe"
-nunit_command  = "packages/NUnit.Runners.2.6.3/tools/nunit-console.exe"
-xunit_command = "packages/xunit.runner.console.2.0.0/tools/xunit.console.exe"
+nuget_command   = ".nuget/nuget.exe"
+gitlink_command = "packages/gitlink.2.2.0/lib/net45/GitLink.exe"
+nunit_command   = "packages/NUnit.Runners.2.6.3/tools/nunit-console.exe"
+xunit_command   = "packages/xunit.runner.console.2.0.0/tools/xunit.console.exe"
 
 solution        = "FakeItEasy.sln"
 assembly_info   = "src/CommonAssemblyInfo.cs"
 version         = IO.read(assembly_info)[/AssemblyInformationalVersion\("([^"]+)"\)/, 1]
 version_suffix  = ENV["VERSION_SUFFIX"]
+repo_url        = "https://github.com/FakeItEasy/FakeItEasy"
 nuspec          = "src/FakeItEasy/FakeItEasy.nuspec"
 analyzer_nuspec = "src/FakeItEasy.Analyzer/FakeItEasy.Analyzer.nuspec"
 logs            = "artifacts/logs"
 output          = "artifacts/output"
 tests           = "artifacts/tests"
 
+gitlink_ignores = [
+  "FakeItEasy.Examples",
+  "FakeItEasy.Tests",
+  "FakeItEasy.IntegrationTests",
+  "FakeItEasy.IntegrationTests.VB",
+  "FakeItEasy.Examples.VB",
+  "FakeItEasy.Specs",
+  "FakeItEasy.IntegrationTests.External",
+  "FakeItEasy.Analyzer.Tests",
+  "FakeItEasy.Tests.Approval"
+]
+

```

Comment:
GitLink 2.3.0 introduces `-include`, so when that goes RTM when can upgrade and list the two assemblies we want to include, instead of listing the 9 we want to ignore.